pkg/config/dynamicconfig.go

```diff
@@ -19,6 +19,10 @@ import (
 // this configmap stores the resources that are to be filtered
 const cmNameEnv string = "INIT_CONFIG"
 
+var ExcludeGroupRule []string
+var ExcludeUsername []string
+var RestrictDevelopmentRole = []string{"minikube-user", "kubernetes-admin"}
```

Comment:
Seems like these 3 exportable vars are consumed by multiple packages - it is updated in pkg/config and read by pkg/engine. It could potentially cause inconsistency across packages. Can we embed a lock and build methods (set / get ) around them?

Reply:
only config package can update them.